cypress/integration/layoutValidation.cy.js

```diff
@@ -1,55 +1,116 @@
-import { CHILD_PROGRAM } from '../data/index.js'
+import { E2E_PROGRAM } from '../data/index.js'
 import {
     clickAddRemoveMainDimension,
     clickAddRemoveProgramDimension,
     openProgramDimensionsSidebar,
+    selectEnrollmentWithProgram,
     selectEventWithProgram,
+    selectTrackedEntityWithType,
 } from '../helpers/dimensions.js'
 import { clickMenubarUpdateButton } from '../helpers/menubar.js'
 import { goToStartPage } from '../helpers/startScreen.js'
 import { expectTableToBeVisible } from '../helpers/table.js'
 
-/* This files constains sequential tests, which means that some test steps
- * depend on a previous step. With test isolation switched on (the default setting)
- * each step (`it` block) will start off in a fresh window, and that breaks this kind
- * of test. So `testIsolation` was set to false here. */
-describe('layout validation', { testIsolation: false }, () => {
-    const trackerProgram = CHILD_PROGRAM
+describe('layout validation', () => {
+    const types = ['event', 'enrollment']
+    types.forEach((type) => {
+        it(`validates that program, columns and org unit are required (${type})`, () => {
+            const trackerProgram = E2E_PROGRAM
 
-    it('program is required', () => {
+            goToStartPage()
+
+            if (type === 'enrollemnt') {
```

Comment:
I think it still works because https://github.com/dhis2/line-listing-app/blob/d509b689e235c3a592071ed0465e6a1fda018cf4/cypress/helpers/dimensions.js#L8 changes it back to "Event" again... anyway, I'll fix the typo